### DISCUSSION TITLE
[photos] Show collect photo uploader name to all users

### DISCUSSION THREAD

**Discussion by Trekky12** [ente-io/ente#4657] [Enhancements]
### Description

When sharing an album with an public link and additional to other users the name of an „anonymous uploader“ (via web) is only visible to the owner of the shared album.
For the other users of the album the owner is shown as uploader.

### What product are you using?

Ente Photos

**Comment by ua741** [comment]
This is by design. That said, we can think about changing the implementation in the future if there’s a good use case and enough user demand.

**Reply by Trekky12** [reply]
I think it could be interesting for other collaborators who contributed which image. Maybe this could also be a flag when sharing an album whether other contributers names should be shown or not.

**Reply by Trekky12** [reply]
If you don’t want to show the exact uploader is it possible to show „anonymous“ instead of the album owner?
Showing the owner of the album leads to confusion e.g. when he gets asked where this photo was taken.

**Comment by ezzra** [comment]
this is unfortunate. Because there is this collaboration feature, that is advertised as
> "A common use case for this is collecting event and trip photos from a big circle of people."

and then uploaders are also asked for their names which is pretty neat and useful. But here it stops, because the other persons from my circle are not able to filter the photos by the uploader or even see the uploader name. This is quite hard to explain to my circle of people I want to invite for sharing photos and motivate in using ente.io
